app/views/admin/users/index.html.haml

```diff
@@ -0,0 +1,19 @@
+%h1 Interviewers
```

Comment:
We can not have part of commits in haml and others in erb.
And haml isn't best option.